data-prepper-plugins/otlp-source/src/main/java/org/opensearch/dataprepper/plugins/source/otlp/OTLPSourceConfig.java

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.otlp;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.Size;
+import org.apache.commons.lang3.StringUtils;
+import org.opensearch.dataprepper.model.types.ByteCount;
+import org.opensearch.dataprepper.plugins.codec.CompressionOption;
+import org.opensearch.dataprepper.plugins.otel.codec.OTelOutputFormat;
+import org.opensearch.dataprepper.plugins.server.RetryInfoConfig;
+import org.opensearch.dataprepper.model.configuration.PluginModel;
+
+public class OTLPSourceConfig {
+  static final String REQUEST_TIMEOUT = "request_timeout";
+  static final String PORT = "port";
+  static final String LOGS_PATH = "logs_path";
+  static final String METRICS_PATH = "metrics_path";
+  static final String TRACES_PATH = "traces_path";
+  static final String SSL = "ssl";
+  static final String LOGS_OUTPUT_FORMAT = "logs_output_format";
+  static final String METRICS_OUTPUT_FORMAT = "metrics_output_format";
+  static final String TRACES_OUTPUT_FORMAT = "traces_output_format";
+  static final String USE_ACM_CERT_FOR_SSL = "useAcmCertForSSL";
```

Comment:
Let's give this a better name:

```
use_acm_certificate_for_ssl
```

It is a new source. We don't need to use the old names. And we use this name elsewhere.